packages/build/tests/error/snapshots/tests.js.md

```diff
@@ -2371,29 +2465,24 @@ Generated by [AVA](https://avajs.dev).
         types:␊
           - node␊
       ts-node:␊
-        cwd: packages/build/tests/error/fixtures/typescript␊
-        projectSearchDir: packages/build/tests/error/fixtures/typescript␊
+        cwd: /tmp-dir␊
+        projectSearchDir: /tmp-dir␊
         require: []␊
-    ␊
-      Plugin details␊
-      Package:        ./plugin.ts␊
-      Version:        1.0.0␊
-      Repository:     git+https://github.com/netlify/build.git␊
-      Report issues:  https://github.com/netlify/build/issues␊
     ␊
       Error location␊
       While loading "./plugin.ts" from netlify.toml␊
     STACK TRACE␊
     ␊
       Error properties␊
       {␊
-        diagnosticText: "plugin.ts(4,25): error TS2339: Property 'DOES_NOT_EXIST' does not exist on type 'NetlifyPluginConstants'./n",␊
-        diagnosticCodes: [ 2339 ]␊
+        diagnosticText: "plugin.ts(1,28): error TS2307: Cannot find module '@netlify/build' or its corresponding type declarations./n" +␊
```

Comment:
Without the fix of this PR, this build error shows `ENOENT` instead.

The build error changed due to the addition of `copyRoot` in the test, which puts the fixtures directory in the global temporary directory instead of as a subdirectory of `netlify/build`. 

This shows another bug related to not being able to find `@netlify/build` types, which is in fact the bug I've been trying to reproduce, as described in https://github.com/netlify/build/issues/3852! So this is good that this test is now showing this bug that I am trying to fix.